util/log/clog.go

```diff
@@ -617,11 +617,10 @@ func Flush() {
 
 // loggingT collects all the global state of the logging setup.
 type loggingT struct {
-	// Boolean flags. Not handled atomically because the flag.Value interface
-	// does not let us avoid the =true, and that shorthand is necessary for
-	// compatibility. TODO: does this matter enough to fix? Seems unlikely.
-	toStderr        bool          // The -logtostderr flag.
-	alsoToStderr    bool          // The -alsologtostderr flag.
+	// Boolean flags.
+	toStderr     bool // The -logtostderr flag. Updated atomically.
```

Comment:
The comment "Updated atomically" in our code usually means that all access to the variable must go through one of the functions in `sync/atomic`. Here we should be more specific about `atomicBool` and the guarantees it provides. It's not clear to me what those are since the flag package goes through `atomicBool` but all other access to these variables has no synchronization at all. Does this help at all? To solve the problem correctly I would think we would need to either make these variables `atomic.Values` containing a `bool` or put the `sync.Mutex` in `loggingT` and use it on all reads.